lib/crawler/accounts/schemas/user_token.ex

```diff
@@ -0,0 +1,69 @@
+defmodule Crawler.Accounts.Schemas.UserToken do
+  use Ecto.Schema
+
+  import Ecto.Query
+
+  alias Crawler.Accounts.Schemas.{User, UserToken}
+
+  @rand_size 32
+
+  @session_validity_in_days 60
+
+  schema "users_tokens" do
+    field(:token, :binary)
+    field(:context, :string)
+    field(:sent_to, :string)
+    belongs_to(:user, User)
+
+    timestamps(updated_at: false)
+  end
+
+  @doc """
+  Generates a token that will be stored in a signed place,
+  such as session or cookie. As they are signed, those
+  tokens do not need to be hashed.
+
+  The reason why we store session tokens in the database, even
+  though Phoenix already provides a session cookie, is because
+  Phoenix' default session cookies are not persisted, they are
+  simply signed and potentially encrypted. This means they are
+  valid indefinitely, unless you change the signing/encryption
+  salt.
+
+  Therefore, storing them allows individual user
+  sessions to be expired. The token system can also be extended
+  to store additional data, such as the device used for logging in.
+  You could then use this information to display all valid sessions
+  and devices in the UI and allow users to explicitly expire any
+  session they deem invalid.
+  """
+  def build_session_token(user) do
```

Comment:
As the current ticket is `Register a new account` and this module is `UserToken` it's used for the Login ticket, should we remove it @vnntsu 

or you can add a TODO to inform that this module will be used on the Login Backend task

Reply:
Added TODO in [a80bd9b](https://github.com/vnntsu/elixir-ic/pull/31/commits/a80bd9b6de86a023125ff3f8364b33f83c8e831e)